gulpfile.js

```diff
@@ -213,17 +217,41 @@ gulp.task('build-theme-files', ['styles']);
 
 //watch and run corresponding task on change, process changed files only
 gulp.task('watch', ['build-all'], () => {
-  gulp.watch(withTheme('./static/images/**/*.*'), ['images']);
-  gulp.watch(withTheme('./static/other/**/*.*'), ['other']);
-  gulp.watch(withTheme('./static/styles/**/*.{css,sass,scss}'), ['styles']);
-  gulp.watch(withTheme('./static/fonts/**/*.*'), ['fonts']);
-  gulp.watch(withTheme(nonBaseScripts), ['scripts']);
-  gulp.watch(withTheme(baseScripts), ['base-scripts']);
-  gulp.watch(withTheme('./static/vendor/**/*.{css,sass,scss}'), ['vendor-styles']);
-  gulp.watch(withTheme('./static/vendor/**/*.js'), ['vendor-scripts']);
-  gulp.watch(['./static/vendor/**/*.*', '!./static/vendor/**/*.js',
-    '!./static/vendor/**/*.{css,sass,scss}'
-  ], ['vendor-assets']);
+
+  let watchOptions = { interval: 1000 };
+  gulp.watch(withTheme('./static/styles/**/*.{css,sass,scss}'), watchOptions, ['styles']);
+  gulp.watch(withTheme('./static/images/**/*.*'), watchOptions, ['images'])
+    .on('change', browserSync.reload);
+  gulp.watch(withTheme(nonBaseScripts), watchOptions, ['scripts'])
+    .on('change', browserSync.reload);
```

Comment:
Reload auch lieber nicht - manchmal habe ich im Fenster einen Stand, den ich während der Entwicklung nicht verlieren will, oder bestimmte Infos in den Devtools - das würde bei der ersten Speicheraktion durch den Browser-Reload verloren gehen. Als Entwickler bestimmte ich lieber selbst das Reloadverhalten.

Reply:
siehe open :-D